packages/ast-position/lib/api.js

```diff
@@ -0,0 +1,12 @@
+const { AstPositionVisitor } = require("./ast-position");
+const { accept } = require("@xml-tools/ast");
+
+function getAstNodeInPosition(ast, offset) {
```

Comment:
This function should also be in the `api.d.ts`. Its return type should be a union type of all the defined interfaces in there and `undefined`.